coalib/bearlib/languages/definitions/PHP.py

```diff
@@ -3,4 +3,20 @@
 
 @Language
 class PHP:
-    pass
+    aliases = 'php'
```

Comment:
Should be a tuple.
ref. https://github.com/coala/coala/blob/ac5aa59e7fa27ce13fca57ed40218477e568307a/coalib/bearlib/languages/Language.py#L178

Reply:
See also `git grep aliases`.
However there is one bug in an existing definition.
https://github.com/coala/coala/issues/5463